### PR TITLE
A note on data privacy

### PR DESCRIPTION
**Pull request by sankarshanmukhopadhyay**
An explanatory note on data privacy related to third-party services being used by the project.

### CODE REVIEW

**Review by amarts — APPROVED** [review]

File: data-privacy-note.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+for project management and community engagement. We are not
+responsible or liable for the privacy standards, practices, and
+policies implemented by such services.
+
+A list of currently used services and their privacy statements or
+policies is provided below to understand how they may collect
+information.
+
+* GItHub: [https://docs.github.com/en/github/site-policy/github-privacy-statement](https://docs.github.com/en/github/site-policy/github-privacy-statement)
+* YouTube: [https://policies.google.com/privacy](https://docs.github.com/en/github/site-policy/github-privacy-statement)
```
Comment (amarts) [inline data-privacy-note.md:11]: wait, the link is all pointing to same github privacy here. Need changes in second section...

Comment (amarts) [inline data-privacy-note.md:11]:
```suggestion
* YouTube: [https://policies.google.com/privacy](https://policies.google.com/privacy)
```

Comment (sankarshanmukhopadhyay) [reply data-privacy-note.md:11]: :) copy paste error. Let me fix it right quick

File: data-privacy-note.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* Zoom: [https://zoom.us/privacy](https://docs.github.com/en/github/site-policy/github-privacy-statement)
```
Comment (amarts) [inline data-privacy-note.md:12]:
```suggestion
* Zoom: [https://zoom.us/privacy](https://zoom.us/privacy)
```

File: data-privacy-note.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+* Google Meet: [https://policies.google.com/privacy?hl=en](https://docs.github.com/en/github/site-policy/github-privacy-statement)
```
Comment (amarts) [inline data-privacy-note.md:13]:
```suggestion
* Google Meet: [https://policies.google.com/privacy](https://policies.google.com/privacy?hl=en)
```

File: data-privacy-note.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+* Discord: [https://discord.com/privacy](https://docs.github.com/en/github/site-policy/github-privacy-statement)
```
Comment (amarts) [inline data-privacy-note.md:14]:
```suggestion
* Discord: [https://discord.com/privacy](https://discord.com/privacy)
```

**Pull request closed without merging**